locale/en/api.po

```diff
@@ -272,6 +272,9 @@ msgstr "The requested volume, number or year is not valid."
 msgid "api.submissions.400.invalidSubmitAs"
 msgstr "You are not allowed to submit in this user role."
 
+msgid "api.submission.403.cantChangeSubmissionLanguage"
```

Comment:
This isn't really a comment about the locale key, but why can't the submission locale be changed if there's more than one publication version? I can think of a few, but I'm wondering if there was a specific technical/editorial process reason for limiting it like this?

Reply:
We realized that having versions would mean lots more to consider, and that the main use case actually happens before the publication. The main use case when this functionality is really needed is when author wrongly chooses the submission locale at the submission. Editors would mostly realize this immediately, and the fix should happen before the actual publication.